jccli/cli/system.py

```diff
@@ -0,0 +1,81 @@
+import json
+import click
+from jccli.jc_api_v1 import JumpcloudApiV1
+
+
+@click.group()
+@click.pass_context
+def system(ctx):
+    """
+    Command set for systems.
+    """
+    pass
+
+
+@system.command("get")
+@click.option('--id', '-i', 'system_id', required=True, type=str)
```

Comment:
how about adding `help='The system ID'`?  would be great to add help to all of the options here and everywhere else.